app/assets/stylesheets/base.scss

```diff
@@ -62,3 +63,20 @@ dd {
 label {
   font-weight: normal;
 }
+
+.btn {
+  background-color: $btn-default-bg;
+  border-color: $btn-default-border;
+  color: $btn-default-color;
+}
+
+.btn-primary {
```

Comment:
Is it possible to use bootstrap mixin to create this button? Or perhaps since the colors are set now, take advantage of the bootstrap overrides in https://github.com/sul-dlss/sul-bento-app/pull/74?

Reply:
If we change the order discussed here https://github.com/sul-dlss/sul-bento-app/pull/74/files#r139425126

Then we can do something like this:

```sass

.btn-sul {
  @include button-variant($cardinal-red, $cardinal-red);
}
```

And get rid of all of this stuff. It would also allow us to get rid of unused variables like `$btn-primary-color` as Bootstrap 4.0.0.beta removed support for those.

Reply:
for now, I changed the `theme-colors` in `bootstrap_variables.scss`. This sets `btn-primary` to `$cardinal-red`. I'm currently wrapping my head around how `button-variant` is used in Bootstrap 3 vs. 4